Tasks/DotNetCoreCLI/pushcommand.ts

```diff
@@ -31,9 +29,8 @@ export async function run(): Promise<void> {
             }
         });
 
-        if (filesList && filesList.length < 1)
-        {
-            tl.setResult(tl.TaskResult.Succeeded, tl.loc("Info_NoPackagesMatchedTheSearchPattern"));
+        if (filesList.length < 1) {
```

Comment:
If it is null, it'll fail in the forEach step itself. findMatch will never return null https://github.com/Microsoft/vsts-task-lib/blob/master/node/task.ts#L1525